src/spec.h

```diff
@@ -678,6 +678,8 @@ size_t IndexSpec_collect_numeric_overhead(IndexSpec *sp);
 /**
  * @return all memory used by the index `sp`.
  * Uses the sizes of the doc-table, tag and text overhead if they are not `0`.
+ * TODO: fIX so this will account for the entire index memory, currently it is a best effort that
+ * account only for part of the actual memory.
```

Comment:
i would say that the future task it get rid of this and have an allocator utility :)